bundles/visualisation_start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+ETCD_HOST=130.89.227.192 ETCD_PORT=4001 java -Dorg.ops4j.pax.logging.DefaultServiceLog.level=OFF -jar visualisation-0.1.jar
+#ETCD_HOST=192.168.1.2 ETCD_POST=4001 java -jar visualisation-0.1.jar
```

Comment:
Can this comment be removed, or is this here to easily switch from local to remote? (Personally I would set the IP in the repo to something generic.)